quartz_solar_forecast/pydantic_models.py

```diff
@@ -25,6 +32,18 @@ class PVSite(BaseModel):
         json_schema_extra=["enphase", "solis", "givenergy", "solarman", None],
     )
 
+    def get_inverter(self):
```

Comment:
If we're only defining data classes then that's fair enough, but if we want to add behaviour to our classes then we're going to be doing some processing. 

Perhaps it would make sense to clearly define what the intention of `pydantic_models.py` is? It looks like for a long time it's only contained the `PVSite` model. Perhaps we should just rename the file `site.py`? The API models that have been added recently would probably be better off living closer to the API code. We don't want to fall into the trap of putting every model in one file because it was named a certain way a while ago.

Reply:
> If we're only defining data classes then that's fair enough, but if we want to add behaviour to our classes then we're going to be doing some processing.
> 
> Perhaps it would make sense to clearly define what the intention of `pydantic_models.py` is? It looks like for a long time it's only contained the `PVSite` model. Perhaps we should just rename the file `site.py`? The API models that have been added recently would probably be better off living closer to the API code. We don't want to fall into the trap of putting every model in one file because it was named a certain way a while ago.

Quite recently the pydantic_models.py file has also accomodated the api models
But yes for the inverter classes at least having separate files within the /inverters directory makes more sense